lib/authorized_persona/authorization.rb

```diff
@@ -41,21 +41,21 @@ def grant(privileges) # rubocop:disable Metrics/AbcSize
         tier_names = authorization_persona.authorization_tier_names
         extra_keys = authorized_actions.keys - authorization_persona.authorization_tier_names
         if extra_keys.present?
-          raise AuthorizedPersona::Error, "invalid grant: #{authorization_persona_class_name} " \
-                                          "has authorization tiers #{tier_names.join(', ')} but received extra keys: #{extra_keys.join(', ')}"
+          raise Error, "invalid grant: #{authorization_persona_class_name} " \
+                       "has authorization tiers #{tier_names.join(', ')} but received extra keys: #{extra_keys.join(', ')}"
```

Comment:
This branch can really only be reached if you set `controller.authorization_current_user_method =` _after_ having already called the `authorize_persona` directive. I wasn't sure if this was really an intended feature, and if not then this `raise` is made redundant by the check on line 19.

However, I erred on the side of leaving lib code in place, rather than making a breaking change to remove (or make private) `#authorization_current_user_method=`.